record/record_test.go

```diff
@@ -399,13 +399,16 @@ func TestBasicRecover(t *testing.T) {
 }
 
 func TestComplexRecover(t *testing.T) {
-	// The first record will be blockSize * 3 bytes long. Since each block has
-	// a 6 byte header, the first record will roll over into 4 blocks.
+	// The first record will be blockSize * 3 bytes long. Since each block has a
```

Comment:
I'm not sure what you mean here. I had a long comment before and you requested I wrap it at 80.  I didn't change this one really. Should I just leave these mid function comments as long as I can and never wrap? Is that what you're asking?